src/components/ChatView.vue

```diff
@@ -19,15 +19,40 @@
   -->
 
 <template>
-	<div class="chatView">
-		<MessagesList :token="token" />
-		<NewMessageForm />
+	<div
+		class="chatView"
+		@dragover.prevent="isDraggingOver = true"
+		@dragleave.prevent="isDraggingOver = false"
+		@drop.prevent="handleDropFiles">
+		<transition name="slide" mode="out-in">
+			<div
+				v-show="isDraggingOver"
+				class="dragover">
+				<div class="drop-hint">
+					<div
+						class="drop-hint__icon"
+						:class="{'icon-upload' : !isGuest, 'icon-error' : isGuest}" />
```

Comment:
I really dislike `icon-error`, I think I would keep the upload icon instead.

Also (similar to the empty content message) I think the icons should be 64*64px
Speaking of empty content view, maybe you can use that code layout too (h2, bigger icon)?

https://github.com/nextcloud/spreed/blob/fa890665b2c5f50ac8189ad2c7d72b8edfef64ad/src/views/NotFoundView.vue#L2-L8

Reply:
64 looked gigantic, I increased to 48